### PR TITLE
Add single json test to ci

### PR DESCRIPTION
**Pull request by rafaelfolco**
Exercise single json test

### CODE REVIEW

**Review by k-rister — APPROVED** [review]
good starting point

File: .github/actions/integration-tests/run-ci-stage1 [hunk 1]
```diff
@@ -61,6 +61,11 @@ fi
 eval set -- "${opts}"
 while true; do
     case "${1}" in
+        --ci-param-mode)
+            shift
+            CI_PARAM_MODE="${1}"
+            shift
+            ;;
```
Comment (k-rister) [inline .github/actions/integration-tests/run-ci-stage1:68]: I would prefer a default set above like all the other parameters.

File: .github/actions/integration-tests/run-ci-stage1 [hunk 2]
```diff
@@ -436,13 +441,24 @@ for userenv in ${CI_ACTIVE_USERENVS}; do
...
+                    # mv-params param mode
+                    run_cmd "crucible run oslat --tags run:1${TAGS} --num-samples ${CI_SAMPLES} --test-order s --mv-params ${SCRIPT_DIR}/mv-params/oslat-1.json ${ENDPOINT_ARG}"
+                fi
                 post_run_cmd
 
                 if [ "${CI_REPEAT_RUNS}" == "yes" ]; then
                     run_cmd "crucible run oslat --tags run:2${TAGS} --num-samples ${CI_SAMPLES} --test-order r --mv-params ${SCRIPT_DIR}/mv-params/oslat-2.json ${ENDPOINT_ARG}"
                     post_run_cmd
                 fi
+
```
Comment (k-rister) [inline .github/actions/integration-tests/run-ci-stage1:461]: delete

File: .github/actions/integration-tests/run-ci-stage1 [hunk 3]
```diff
@@ -436,13 +441,24 @@ for userenv in ${CI_ACTIVE_USERENVS}; do
...
+                else
+                    # mv-params param mode
+                    run_cmd "crucible run oslat --tags run:1${TAGS} --num-samples ${CI_SAMPLES} --test-order s --mv-params ${SCRIPT_DIR}/mv-params/oslat-1.json ${ENDPOINT_ARG}"
+                fi
                 post_run_cmd
 
                 if [ "${CI_REPEAT_RUNS}" == "yes" ]; then
                     run_cmd "crucible run oslat --tags run:2${TAGS} --num-samples ${CI_SAMPLES} --test-order r --mv-params ${SCRIPT_DIR}/mv-params/oslat-2.json ${ENDPOINT_ARG}"
                     post_run_cmd
                 fi
```
Comment (k-rister) [inline .github/actions/integration-tests/run-ci-stage1:460]: Shouldn't these be in the else block?

File: .github/workflows/crucible-ci.yaml [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
-    - '**.md'
-    - .github/workflows/faux-crucible-ci.yaml
-    - 'docs/**'
+  #pull_request:
+  #  branches: [ main ]
+  #  paths-ignore:
+  #  - LICENSE
+  #  - '**.md'
+  #  - .github/workflows/faux-crucible-ci.yaml
+  #  - 'docs/**'
```
Comment (k-rister) [inline .github/workflows/crucible-ci.yaml:10]: I assume this is debug?

File: .github/actions/integration-tests/run-ci-stage1 (hunk 1 above)
Comment (rafaelfolco) [reply .github/actions/integration-tests/run-ci-stage1:68]: ack

File: .github/actions/integration-tests/run-ci-stage1 (hunk 3 above)
Comment (rafaelfolco) [reply .github/actions/integration-tests/run-ci-stage1:460]: done

File: .github/workflows/crucible-ci.yaml (hunk 4 above)
Comment (rafaelfolco) [reply .github/workflows/crucible-ci.yaml:10]: yes, debug only

**Comment by rafaelfolco** [comment]
moved to https://github.com/perftool-incubator/crucible-ci/pull/66

**Pull request closed without merging**